Student/FindStudentViewController.swift

```diff
@@ -83,33 +74,50 @@ class findStudentViewController: UIViewController,UITextFieldDelegate {
                 foundStudentPhoneNo.text=student.phoneNo
                 
             }
+            
+        }*/
+        let tempStudent = collegeDb.searchStudentWithRollNo(rollNo: Int(rollID.text!)! )
+        guard let temp=tempStudent else
```

Comment:
use if let in this case. guard is used for early exits only and any code should be avoided in this